run.sh

```diff
@@ -97,11 +97,15 @@ if [ -n "$STATESYNC_RPC_SERVERS" ]; then
   fi
 fi
 
+
 # Initialise
 if [ ! -d "$PROJECT_HOME/config" ]; then
   $PROJECT_BIN init "$MONIKER" --chain-id "$CHAIN_ID"
 fi
 
+# Overwrite seeds in config.toml for chains that are not using the env variable correctly
+sed -i "s/seeds = \"\"/seeds = \"$P2P_SEEDS\"/" $PROJECT_HOME/config/config.toml
```

Comment:
Does Thornode work with the `THOR` namespace set in the docker-compose, just doesn't pick up the seeds? Could we do this step as a custom build step, like Juno? Also see https://github.com/ovrclk/cosmos-omnibus/pull/72

Reply:
This cannot be put during the build time as the config.toml file is created only after the previous step that does the initialization step. I would say this was safe to do for all chains, however I added a new variable that allows overwriting the seeds in the config.toml file.

Reply:
I also tried with THOR_P2P_SEEDS but it doesn't seem to be working. The chain hangs in that case on the initialization phase. It won't fetch any transaction blocks. Overwriting them in the config.toml file seems to be the only working option.